pennylane/devices/qubit/simulate.py

```diff
@@ -57,16 +57,26 @@ def simulate(circuit: qml.tape.QuantumScript, rng=None, debugger=None) -> Result
 
     state = create_initial_state(circuit.wires, circuit._prep[0] if circuit._prep else None)
 
+    # initial state is batched only if the state preparation (if it exists) is batched
+    is_state_batched = False
```

Comment:
Non-blocking comment but I'm not personally a big fan of naming the variable `is_state_batched`. I think a slightly more concise name would be `batched_state`. My personal opinion is to usually avoid using more than one underscore in a variable name. Same thing for the function arguments below.

Reply:
Just linking back to this discussion: https://github.com/PennyLaneAI/pennylane/pull/3852#discussion_r1124951529 .

This is all fairly internal, so I think we'd be fine to change this name around freely. At least for now.  But I think this is a different discussion we can kick down the road.

Reply:
that kwarg was part of 0.30 in `apply_operation.py` and I don't really wanna deprecate it. It wasn't in any public docs... but still

PS: check the user who [committed the variable name](https://github.com/PennyLaneAI/pennylane/commit/1920f9da1939555e99eb29232be6f6e68c9e9ad6) 😆

Reply:
oops too slow

Reply:
This is not super high priority, and according to the linked discussion the name `is_state_batched` satisfies most people. So I'd like to keep it as is at least for now.